packages/gestalt/src/icons/index.js

```diff
@@ -300,6 +302,8 @@ const icons = Object.freeze({
   'arrow-circle-forward': arrowCircleForward,
   'arrow-circle-up': arrowCircleUp,
   'arrow-counter-clockwise': arrowCounterClockwise,
+  'arrow-double-forward': arrowDoubleForward,
```

Comment:
OK, if we are adding icons 

Can we do this on a separate PR first?

We need to also update 

1. TS file for icons
2. We need to also add it to the Android folder.
3. We need to update the visual snapshot for icons.

I think a separate PR would be nice for this.